docs/book/index.md

```diff
@@ -1,25 +1,22 @@
 ---
-description: Example Landing Page
+description: Start here with the ZenML Docs.
 ---
 
 # ZenML
 
-**ZenML** is an extensible, open-source MLOps framework for creating
-production-ready machine learning pipelines. Built
-for data scientists, it has a simple, flexible syntax, is **cloud-** and 
+**ZenML** is an extensible, open-source MLOps framework for creating 
+portable, production-ready MLOps pipelines. Built 
+for data scientists, ML Engineers, and MLOps Developers to collaborate, 
+it has a simple, flexible syntax, is **cloud-** and 
 **tool-agnostic**, and has interfaces/abstractions that are catered towards 
-ML workflows.
+ML workflows. 
 
 At its core, **ZenML pipelines execute ML-specific workflows** from sourcing
-data to splitting, preprocessing, training, all the way to the evaluation of 
-results and even serving. There are many built-in features to support
+data to splitting, preprocessing, training, all the way to serving and monitoring 
+ML models in production. There are many built-in features to support
 common ML development tasks. ZenML is not here to replace the great tools that
-solve these individual problems. Rather, it
-**integrates natively with popular ML tooling** and gives standard
-abstraction which allow you to build your own integration for other tools or 
-your own on-prem solution.
-
-
+solve these individual problems. Rather, it offers an **extensible framework** and a
+standard abstraction to write and build your workflows.
```

Comment:
Whatever we decide for the README text (see above) should be replicated here. (And on the website, if it's being used there as well).